culture/review_culture.md

```diff
@@ -140,3 +140,20 @@ doing this, you should ask: in what ways is doing it my way materially better?
 What benefits come from the changed approach? If you don't have clear answers
 to these questions, don't ask for the change.
 
+## Bias toward "nit-and-approve"
+
+If your review comments primarily suggest local, targeted changes to the
+proposed change, rather than large structural or directional changes, consider
+approving the change on your first review at the same time as leaving your
+comments. This implies that the author of the change should feel free to merge
+the change once they've addressed your concerns. This is especially useful when
+the comments are uncontroversial (typo fixes, small improvements), because the
+reviewer doesn't have to wait for you to take another look at the change and
+"officially" approve it before they can merge it and move on.
+
+Note that this requires a degree of trust. A reviewer should merge a change
+only once they believe in good faith that all comments really have been
+resolved, and that the author of each comment thread would not be surprised by
+how the feedback was received and incorporated. As a change author, if you
+prefer not to incorporate some suggestion that wasn't intended to be optional,
+you should probably continue the discussion rather than merging immediately.
```

Comment:
In terms of author do's: I would strongly encourage to comment about how you've responded to (or left out) the nits.

And think we should touch on that. Either in this section, or as a new section and refer to it. Because it's especially important for "nit-and-approve".

Could be very small, like for a typo just `fixed!` will do.
If it's a larger change, something like this can work:
`Thanks, I've taken a slight variation of your suggestion and merged`
Or when skipping: `This resolves itself in another PR` or `I think it's OK as-is, leaving unchanged`.

When your resolution requires any more detail than that, it's probably better to wait for the reviewer :smiley:

I think this helps to:
- Recognize the reviewer
- Make sure you haven't missed anything
- Add intuition when you should wait for the reviewer (when the comment becomes a spotty justification)
- Add more context for the public archive

Reply:
I agree that it's a best practice to respond to all the nits (whether addressing or leaving unchanged).

Reply:
Okay; updated the text.